addons/test/autoCorrectedMmddyyyyPipe.spec.js

```diff
@@ -28,8 +28,4 @@ describe('autoCorrectedMmddyyyyPipe', () => {
   it('returns false if day 1st digit is 0 and 2nd digit is also 0', () => {
     expect(autoCorrectedMmddyyyyPipe('12/00')).to.equal(false)
   })
-
-  it('completes the year to `200` if the 1st digit is 0', () => {
```

Comment:
Hey, thanks for the PR and for cleaning up this code!!

One issue: I don't understand what you mean by this can be added to the mask easily? If the mask allows, say, `1980` to `present`, then the user can't hardcode `200` in the mask. 

I know that some users currently like the `0` to `200` autocomplete feature.